src/jsonValidation/reminderSettingsValidation.ts

```diff
@@ -29,14 +29,17 @@ const isValidReminderSetting = (data: unknown): data is ReminderSettingJson => {
     zip
   } = data as ReminderSettingJson;
 
+  const cityString = city ?? '';
+  const zipString = zip ?? '';
```

Comment:
naming a variable with a type is generally bad practice. why is this needed for validation?
can't we ensure the correct value without touching the validation?

Reply:
I added a new method to update the data after pulling it from the server. In this method, if the city and zip values are null, they are returned to empty string and sent to validation in that form.

https://github.com/smart-village-solutions/smart-village-app-app/commit/13ebc43362b441c180cd3ca313b2a44efe3c9bb6

Reply:
yeah! i like that ver much more